index.js

```diff
@@ -91,26 +101,39 @@ async function init() {
 		const parsedUrl = new URL(query.get('url'));
 		[, user, repository, ref, dir] = urlParserRegex.exec(parsedUrl.pathname);
 
-		console.log('Source:', {user, repository, ref, dir});
+		isGHE = parsedUrl.hostname !== 'github.com';
+		if (isGHE) {
+			domain = parsedUrl.hostname;
+			apiEndpoint = `${parsedUrl.protocol}//${parsedUrl.host}/api/v3`;
+
+			document.querySelector('#create-token').href = `${parsedUrl.protocol}//${parsedUrl.host}/settings/tokens/new?description=Download GitHub directory&scopes=repo`;
```

Comment:
```suggestion
			document.querySelector('#create-token').host = parsedUrl.host;
```

Reply:
I did that because I wasn't sure if GitHub Enterprise was installable with only `http`. Do you think we can just ignore that case?

Reply:
Yes.